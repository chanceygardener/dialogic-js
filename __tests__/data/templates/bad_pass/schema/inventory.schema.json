{
	"name": "inventory",
	"schema": {
		"InventoryQuery": {
			"args": {
				"num_items": {
					"type": "num",
					"description": "The number of specified items that can be found in the inventory",
					"required": true
				},
				"item_singular": {
					"type": "str",
					"description": "the singular form of the item in question",
					"required": false
				},
				"item_plural": { 
						"type":"str",
						"description": "the plural form of the item in question",
						"required": true
				}
			},
			"description": "The user is asking how many of a certain item exists in the inventory."
		},
		"OpenHoursBound": {
			"args": {
				"refer_open": {
					"type": "bool",
					"description": "True if the user is asking the question referring to the opening time, false if they are referring to the closing time",
					"required": true
				},
				"time": {
					"type": "str",
					"description": "The time the store is opening, expressed as a string",
					"required": true
				},
				"date": {
					"type": "str",
					"description": "The date in question, expressed as a string",
					"required": true
				},
				"is_certain": {
					"type": "bool",
					"description": "True if the agent has confirmation that this is the case today, otherwise we assume the schedule is inferred by 'usual' hours, and include a disclaimer in the response text",
					"required": true
				}
			}
		}
	}
}