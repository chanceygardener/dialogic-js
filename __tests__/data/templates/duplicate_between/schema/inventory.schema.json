{
	"name": "inventory",
	"schema": {
		"InventoryQuery": {
			"args": {
				"num_items": {
					"type": "num",
					"description": "The number of specified items that can be found in the inventory",
					"required": true
				},
				"item_singular": {
					"type": "str",
					"description": "the singular form of the item in question",
					"required": false
				},
				"item_plural": { 
						"type":"str",
						"description": "the plural form of the item in question",
						"required": true
				}
			},
			"description": "The user is asking how many of a certain item exists in the inventory."
		}
	}
}