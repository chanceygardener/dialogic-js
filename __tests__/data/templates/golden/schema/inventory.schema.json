{
	"name": "inventory",
	"schema": {
		"ImportTest": {
			"args": {},
			"description": "This template simply calls another that has been imported from the sample domain. It's here to test the functionality of the import statement"
		},
		"ImportTestToFail": {
			"args": {},
			"description": "This template simply calls another that has not been imported from the sample domain. It's here to test the functionality of the import statement"
		},
		"InventoryQuery": {
			"args": {
				"num_items": {
					"type": "num",
					"description": "The number of specified items that can be found in the inventory",
					"required": true
				},
				"item_singular": {
					"type": "str",
					"description": "the singular form of the item in question",
					"required": false
				},
				"item_plural": { 
						"type":"str",
						"description": "the plural form of the item in question",
						"required": true
				}
			},
			"description": "The user is asking how many of a certain item exists in the inventory."
		}
	}
}